Gemfile

```diff
@@ -1,9 +1,8 @@
 source 'https://www.rubygems.org'
+ruby '~> 2.7'
 
-ruby "~> 2.7"
-
-gem 'middleman'
 gem 'builder'
-gem 'pry'
 gem 'git'
+gem 'middleman'
```

Comment:
This is very nitpicky, but could we leave `middleman` at the top as it's really the most important one.  I think rubocop will preserve gaps between chunks of gem definitions and not smoosh it all into a single list.

Reply:
Updated :)

I've developed a preference to sort alphabetically without spaces or comments, it's come from the process of being able to run `bundle add GemName && sortGemfileCommand` :)

Reply:
huh, I have _never_ `bundle add GemName`.  TIL!